read-fonts/src/tables/vmtx.rs

```diff
@@ -0,0 +1,6 @@
+//! The [vmtx (Vertical Metrics)](https://docs.microsoft.com/en-us/typography/opentype/spec/vmtx) table
+
+use font_types::Tag;
+
+/// 'vmtx'
+pub const TAG: Tag = Tag::new(b"vmtx");
```

Comment:
okay, I have an idea for how we might want to structure this:

Here's what I think I would do:

- make the `hvmtx` and `hvhea` modules marked `pub(crate)` in `tables.rs`.
- manually reexport any shared structs (`LongMetric` in hvmtx, e.g.) from here, into each of the `hmtx` and `vmtx` modules
- add `pub` type aliases for `Hhea` and `Vhea` in the appropriate modules, with the appropriate doc strings

So in this file we would add something like,

```rust

pub use super::hvmtx::LongMetric;

/// The [vmtx (Vertical Metrics)](https://docs.microsoft.com/en-us/typography/opentype/spec/vmtx) table
pub type Vmtx = super::hvmtx::HVmtx;
```

and do this same trick in the others.

Reply:
I can do that and add those type aliases, but what should methods like hhea() return, the shared `HVhea` struct or the type-aliased `Hhea` one?

```diff
- fn hhea(&self) -> Result<tables::hvhea::HVhea<'a>, ReadError>
+ fn hhea(&self) -> Result<tables::hhea::Hhea<'a>, ReadError>
```

Wouldn't we then basically hide from the API the fact that those tables are actually binary-encoded the same way and one can expect to use them interchangeably in user code?

Reply:
that's a nice point. The final answer will depend on what we resolve for #109. let's leave this as is (no type aliases) for now, and we can revisit once we know our plan for the tag trait.